perprof/main.py

```diff
@@ -186,6 +187,8 @@ def set_arguments(args):
                     'is no primal feasibility to check.'))
     parser.add_argument('--infeasibility-tolerance', type=float, default=1e-4,
             help=_('Tolerance for the primal and dual infeasibilities'))
+    parser.add_argument('--title', type=str,
+            help=_('Sets the performance profile title'))
```

Comment:
I prefer something like "Set the title to be show on top of the performance profile".